src/blocks/_fhi-search.block.scss

```diff
@@ -19,26 +19,24 @@
 
     &__btn {
         @extend .ms-1;
+        @include fhi-btn-dark;
         border-radius: $fhi-px * 22;
+        flex-shrink: 0;
+        text-overflow: unset;
         white-space: nowrap;
 
-        &,
-        &:hover,
-        &:focus,
-        &:active,
-        &:focus:hover {
-            font-family: $fhi-font-family-sans-serif-regular;
+        &:focus {
             text-decoration: none;
-        }
 
-        &:focus-visible {
             .fhi-search__btn-icon {
                 filter: invert(100);
-
-                // TODO: remove data-whatintent when we get rid of the programmatic focus logic
-                [data-whatintent="mouse"] & {
-                    filter: invert(0);
-                }
+            }
+        }
+        &:hover {
+            text-decoration: none;
+            
```

Comment:
Is it intentional that hover effect is removed?

Reply:
Hover state was not stated in Figma, but is now cleared with designer.